osu.Game/Screens/Select/FilterControl.cs

```diff
@@ -161,6 +162,14 @@ private void load(OsuColour colours, IBindable<RulesetInfo> parentRuleset, OsuCo
                                         Origin = Anchor.TopRight,
                                         RelativeSizeAxes = Axes.X,
                                         Width = 0.4f,
+                                    },
+                                    beatmapCountText = new OsuSpriteText
+                                    {
+                                        Text = "Test",
```

Comment:
This sort of "test" stuff cannot land in an actual pull request. Please audit your changes before submitting.